src/xenia/cpu/ppc/ppc_emit_memory.cc

```diff
@@ -674,13 +691,19 @@ int InstrEmit_stdbrx(PPCHIRBuilder& f, const InstrData& i) {
 // Integer load and store multiple (A-16)
 
 int InstrEmit_lmw(PPCHIRBuilder& f, const InstrData& i) {
-  XEINSTRNOTIMPLEMENTED();
-  return 1;
-}
+  Value* b;
 
-int InstrEmit_stmw(PPCHIRBuilder& f, const InstrData& i) {
-  XEINSTRNOTIMPLEMENTED();
-  return 1;
+  for (uint32_t j = 0; j < 32 - i.D.RT; ++j) {
```

Comment:
`if (i.DS.RT + j == i.DS.RA) { continue; }`, according to the example with -1 and the comment about `lm` on POWER in the documentation? But I'm not sure about this.

Reply:
The second part of the if statement satisfies the condition of that example. if we try to load something from GPR 30 into GPR 30, it will not actually have an effect. However, your suggested change is an optimization to this process.